pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/pom.xml

```diff
@@ -100,4 +98,28 @@
       <artifactId>jopt-simple</artifactId>
     </dependency>
   </dependencies>
+
+
+  <profiles>
+    <profile>
+      <id>scala-2.12</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
This basically means the default scala version got downgraded from 2.13 to 2.12 in the project. Could you elaborate a bit more on the reason of doing that to the description?

Reply:
The reason is I've got a lot request for scala 2.12 support, hence make this as default and allow upgrade to 2.13.

Basically this is similar to kafka 0.9 and 2.x, users can build a connector based on the scala version from their enviroment.

Reply:
For Pinot users, there is no different in terms of using scala 2.12 or 2.13 built jars.
This is mostly for users who are integrating Pinot code and libs and making their life easier.